### PR TITLE
SNTV rule

### PR DESCRIPTION
**Pull request by Janix4000**

### CODE REVIEW

**Comment by Janix4000** [comment]
[Implement SNTV rule](https://trello.com/c/1lWmgIcp/6-implement-sntv-rule)

File: votepy/sntv.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    Args:
+        voting (OrdinalElection): voting for which the function calculates the committee
+        size_of_committee (int): Size of the committee
+
+    Raises:
+        ValueError: Size of commite is a positive number which do not exceeds number of all candidates
+        ValueError: Number of candidates scored in SNTV is a positive number which do not exceeds number of all candidates
+
+    Returns:
+        OrdinalBallot: List of chosen candidates wrapped in ordinalBallot
```
Comment (Wolokin) [inline votepy/sntv.py]: It returns a list, not an OrdinalBallot

**Review by Wolokin — CHANGES_REQUESTED** [review]
Small tweak needed

File: votepy/sntv.py (hunk 1 above)
Comment (Janix4000) [reply votepy/sntv.py]: Fixed

**Review by Wolokin — APPROVED** [review]
Ok

**Pull request merged**